app/src/main/java/mobi/visamate/MainActivity.kt

```diff
@@ -37,6 +43,14 @@ class MainActivity : AppCompatActivity() {
                     .setAction("Action", null).show()
         }
 
+
+        //TODO move this to a unit test and use it on another Activity/Fragment for Contributors
+        val mLiveDataTimerViewModel = ViewModelProviders.of(this).get(ContributorsViewModel::class.java!!)
```

Comment:
Starting with a first polemic comment: are we using Hungarian Notation? ;)

Reply:
Feel free to use it or not